test/App.js

```diff
@@ -0,0 +1,15 @@
+const { expect } = require('chai');
```

Comment:
Add the following two lines to import app object:
```
var app = require('../App.js')
var request = require('supertest')
```

Reply:
will fix it.